### PR TITLE
feat: simplify configuration

### PR DESCRIPTION
**Pull request by louisgrasset**
This PR brings the following changes:

### 📚 Documentation
* Update all the runtimes doc
* Migration of all doc files to the doc center: https://louisgrasset.github.io/touitomamout/docs 

### ⚙️ Configuration simplification
* removed the `EXECUTION` notion, manual sync will not rely on the deploy.sh script anymore. So the PM2 runtime will not need a specific flag too.
* removed the `INSTANCE_ID` notion, it will now be automatically handled internally.
* simplified the deployment script and rename it to `pm2.sh`.
* made twitter handle more robust by trimming its value, removing '@' and so on.
* remove the `DAEMON_PERIOD_MIN` variable.
* added the `SYNC_FREQUENCY_MIN` variable; used with the new `DAEMON` variable, it allows docker & manual runtimes to regularly run sync.

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -16,26 +16,38 @@ if (!envAvailable) {
   throw new Error("No suitable .env file found.");
 }
 
+const trimTwitterHandle = (handle: string) => {
+  return handle.toLowerCase().trim().replaceAll(" ", "_").replaceAll("@", "");
```
Comment (ralmn) [inline src/constants.ts:20]: Why turn space in underscore ?

File: src/constants.ts [hunk 2]
```diff
@@ -16,26 +16,38 @@ if (!envAvailable) {
...
+  process.env.TWITTER_USERNAME || "",
+);
+export const TWITTER_PASSWORD = (process.env.TWITTER_PASSWORD || "").trim();
+export const MASTODON_INSTANCE = (process.env.MASTODON_INSTANCE || "").trim();
+export const MASTODON_ACCESS_TOKEN = (
+  process.env.MASTODON_ACCESS_TOKEN || ""
+).trim();
+export const BLUESKY_INSTANCE = (process.env.BLUESKY_INSTANCE || "").trim();
+export const BLUESKY_IDENTIFIER = (process.env.BLUESKY_IDENTIFIER || "").trim();
+export const BLUESKY_PASSWORD = (process.env.BLUESKY_PASSWORD || "").trim();
```
Comment (ralmn) [inline src/constants.ts:38]:
```suggestion
export const BLUESKY_INSTANCE = process.env.BLUESKY_INSTANCE?.trim() || "";
export const BLUESKY_IDENTIFIER = process.env.BLUESKY_IDENTIFIER?.trim() "";
export const BLUESKY_PASSWORD = process.env.BLUESKY_PASSWORD?.trim() || "";
```

File: src/constants.ts [hunk 3]
```diff
@@ -16,26 +16,38 @@ if (!envAvailable) {
...
+  process.env.TWITTER_HANDLE || "",
+);
+export const TWITTER_USERNAME = trimTwitterHandle(
+  process.env.TWITTER_USERNAME || "",
+);
+export const TWITTER_PASSWORD = (process.env.TWITTER_PASSWORD || "").trim();
+export const MASTODON_INSTANCE = (process.env.MASTODON_INSTANCE || "").trim();
+export const MASTODON_ACCESS_TOKEN = (
+  process.env.MASTODON_ACCESS_TOKEN || ""
+).trim();
```
Comment (ralmn) [inline src/constants.ts:35]:
```suggestion
export const MASTODON_ACCESS_TOKEN = process.env.MASTODON_ACCESS_TOKEN?.trim() || "";
```

File: src/constants.ts (hunk 2 above)
Comment (louisgrasset) [reply src/constants.ts:38]: To keep consistency between all consts, I prefer defining the value and _then_ trim it. But sure, it's not necessary here

**Pull request merged**